### ISSUE TITLE
Migrate remaining telemetry related to autocomplete domains

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/focus-android#5885]
Breakout from #5488.

> Questions being answered by the data:

Are users customizing the autocompletes?
Are users wanting to quickly get to websites that they frequent?

**Comment by lobontiumira** [comment]
This is not fixed yet. I was not able to trigger these events.
I've enabled/disabled the autocomplete toggles for top sites, and for sites you add, and these modifications did not trigger any event/metric.

Tested on 11/29 Focus Nightly build with Google Pixel (Android 10).

**Comment by mcarare** [comment]
@lobontiumira Sorry, the PR that contains the commit related to this has not been merged yet. Will add the label when it is ready for QA.

**Comment by lobontiumira** [comment]
Verified as fixed on the latest Focus Nightly 97.0 from 12/22 with Google Pixel (Android 10).
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/mira-2112220001)

* top_sites_setting_changed
```
{
          "timestamp": 10088,
          "category": "autocomplete",
          "name": "top_sites_setting_changed",
          "extra": {
            "is_enabled": "false"
          }
        }
```
* favorite_sites_setting_changed
```
{
          "timestamp": 10872,
          "category": "autocomplete",
          "name": "favorite_sites_setting_changed",
          "extra": {
            "is_enabled": "false"
          }
```
